src/components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,76 @@
+import { Avatar, Box, Button, TextField } from '@mui/material'
+import { useForm } from 'react-hook-form'
+import { useNavigate } from 'react-router-dom'
+import { loginFormSchema } from '../../validations/LoginFormSchema'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { useEffect, useState } from 'react'
+import { useSnackbar } from 'notistack'
+
+export interface ILoginForm {
+  username: string
+  password: string
+}
+
+const LoginForm = () => {
+  const navigate = useNavigate()
```

Comment:
El useNavigate() tiene un contrato useNavigate('some string') 
```suggestion
  const navigate = useNavigate('/admin')
```